pom.xml

```
@@ -53,16 +53,16 @@
         <version.org.eclipse.microprofile.lra>2.0</version.org.eclipse.microprofile.lra>
         <version.io.micrometer>1.12.4</version.io.micrometer>
         <version.org.jboss.arquillian>1.7.2.Final</version.org.jboss.arquillian>
-        <version.org.jboss.wildfly.dist>32.0.1.Final</version.org.jboss.wildfly.dist>
+        <version.org.jboss.wildfly.dist>34.0.0.Beta1</version.org.jboss.wildfly.dist>
         <version.org.wildfly.arquillian>5.0.1.Final</version.org.wildfly.arquillian>
         <version.org.fusesource.jansi>1.18</version.org.fusesource.jansi>
         <version.io.undertow.undertow-core>2.3.0.Alpha2</version.io.undertow.undertow-core>
         <version.org.jboss.threads.jboss-threads>2.4.0.Final</version.org.jboss.threads.jboss-threads>
         <version.org.eclipse.microprofile.jwt.microprofile-jwt-auth-api>2.1</version.org.eclipse.microprofile.jwt.microprofile-jwt-auth-api>
         <!-- wildfly-core update needed to have the launcher version with BootableJarCommandBuilder used to start
         the server up -->
-        <version.org.wildfly.core>24.0.1.Final</version.org.wildfly.core>
-        <version.org.wildfly.extras.creaper>2.0.2</version.org.wildfly.extras.creaper>
+        <version.org.wildfly.core>26.0.0.Beta5</version.org.wildfly.core>
```

corresponding core version for WF 34 Final
```suggestion
        <version.org.wildfly.core>26.0.1.Final</version.org.wildfly.core>
```

---

Thanks, fixed.